portfolio/src/main/webapp/login.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="src/styles.css" />
+
+    <!-- Import Bootstrap CSS  -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor"
+      crossorigin="anonymous"
+    />
+
+    <!-- Import Google Fonts: Playfair Display  -->
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Cinzel:wght@500&family=Gowun+Dodum&family=Playfair+Display:wght@400;500&family=Prata&family=Source+Sans+Pro:wght@200&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
+
+	<section class="nav-intro" style="background-image: url('flower.jpg');">
+		<div class="container">
+			<nav class="navbar navbar-expand-md navbar-light pt-5 pb-4">
+					<div class="container-xxl">
+						<!-- navbar brand / title -->
+						<a class="navbar-brand" href="index.html">
+							<span class="fw-bold brand"> Logo</span>
+						</a>
+						<!-- toggle button for mobile nav -->
+						<button
+							class="navbar-toggler"
+							type="button"
+							data-bs-toggle="collapse"
+							data-bs-target="#main-nav"
+							aria-controls="main-nav"
+							aria-expanded="false"
+							aria-label="Toggle navigation"
+						>
+							<span class="navbar-toggler-icon"></span>
+						</button>
+
+						<!-- navbar links -->
+
+							
+						<div
+							class="collapse navbar-collapse justify-content-end align-center"
+							id="main-nav"
+						>		
+							<ul class="navbar-nav">
+								<li class="nav-item">
+									<div class="search">
+										<i class="fa fa-search"></i>
+										<input type="text" class="form-control" placeholder="Search Classes">
+									</div>
+								</li>
+
+								<li class="nav-item">
+											
+								</li>
+
+
+								<li class="nav-item">
+									<a class="nav-link drop" href="#">HOME</a>
+								</li>
+		
+								<li class="nav-item">
+									<a class="nav-link drop" href="#">CLASSES</a>
+								</li>
+	
+								<li class="nav-item">
+									<a class="nav-link drop" href="#">ABOUT</a>
+								</li>
+
+								<li class="nav-item">
+									<a class="nav-link drop" href="#">SIGN IN</a>
+								</li>
+
+							</ul>
+						</div>
+					</div>
+			</nav>
+		</div>
+		<!--Navigation Bar-->
+
+
+		
+    <section class = "Form my-4 mx-5">
+			<div class = "container">
+				<div class = "row no-gutters">
+					<div class = "col-lg-5">
+						<img src = "2345127.jpg" class = "img-fluid" alt = ""/>
```

Comment:
it seems like i dont see any images with this link? But the structures of the website looks fantastic!!!!